checkAndCleanupNoncurrent.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -e
+
+function usage {
+    echo $@ >&2
+    cat >&2 <<EOF
+Usage:
+    checkAndCleanupNoncurrent.sh
+
+Example:
+    docker run -i --net=host --rm zenko/s3utils:latest /usr/bin/env \\
+    S3_ENDPOINT=http://10.97.84.244:8000 \\
+    ACCESS_KEY=ABCDEFGHIJKLMNOPQRST \\
+    SECRET_KEY=abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMN \\
+    BUCKETD_HOSTPORT=10.102.18.103:9000 \\
+    SPROXYD_HOSTPORT=10.109.250.118:8181 \\
+    BUCKETS=bucket1,bucket2 \\
+    ./checkAndCleanupNoncurrent.sh
+
+Mandatory Environment Variables:
+    S3_ENDPOINT: S3 endpoint URL
+    ACCESS_KEY: S3 account/user access key
+    SECRET_KEY: S3 account/user secret key
+    BUCKETD_HOSTPORT: ip:port of bucketd endpoint
+    SPROXYD_HOSTPORT: ip:port of sproxyd endpoint
+    BUCKETS: comma-separated list of buckets to scan, repair and cleanup
+
+Optional Environment Variables:
+    WORKERS: maximum concurrency for metadata checks (default 100)
+    VERBOSE: set to 1 for more verbose output of verify script
+EOF
+    exit 1
+}
+
+if [ "x${S3_ENDPOINT}" = x ]; then
```

Comment:
would testing if the string is empty be simpler?
```suggestion
if [  -z "${S3_ENDPOINT}" ]; then
```

Reply:
Indeed, it's cleaner

Reply:
done